engine/apps/user_management/migrations/0022_alter_team_unique_together.py

```diff
@@ -0,0 +1,60 @@
+# Generated by Django 4.2.11 on 2024-05-24 16:22
+import logging
+
+from django.db import migrations, models
+import django_migration_linter as linter
+
+logger = logging.getLogger(__name__)
+
+
+def clean_up_duplicated_teams(apps, schema_editor):
+    Team = apps.get_model("user_management", "Team")
+
+    # get (organization_id, team_id) pairs for duplicated teams
+    duplicate_rows = Team.objects.values_list(
+        "organization_id", "team_id"
+    ).annotate(count=models.Count("id")).filter(count__gt=1)
+
+    for organization_id, team_id, _ in duplicate_rows:
+        # keep first team
+        first_team = Team.objects.filter(
+            organization_id=organization_id,
+            team_id=team_id,
+        ).order_by("id").first()
+
+        # migrate resources associated to duplicated entries
+        duplicated_teams = Team.objects.filter(
+            organization_id=organization_id,
+            team_id=team_id,
+        ).exclude(id=first_team.id)
+
+        for team in duplicated_teams:
```

Comment:
Do we need to handle user.current_team values here?

Reply:
Hm.. I think we are not using `current_team` anymore? I can do the update here in any case (otherwise `current_team` will be reset to `None` for those users with a deleted current_team, which afaict shouldn't break anything either)